Tasks/choco/Choco.ps1

```diff
@@ -42,9 +40,15 @@ function Ensure-Chocolatey
     if (-not (Test-Path "$ChocoExePath"))
     {
         Set-ExecutionPolicy Bypass -Scope Process -Force
-        $installScript = (New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1')
-        $expression = "$installScript | powershell.exe -NoProfile -ExecutionPolicy Bypass -Command -"
-        Execute -Expression $expression
+        $installScriptPath = Join-Path $env:TEMP "Choco-Install.ps1"
```

Comment:
It would be good to randomize the script path name. For example:
```powershell
$installScriptPath = [System.IO.Path]::GetTempFileName() + ".ps1"
```